package.json

```
@@ -18,6 +18,8 @@
   "author": "Aliaksei Hrynko (https://github.com/hrynko)",
   "repository": "github:hrynko/vue-pdf-embed",
   "scripts": {
+    "prepare": "npm run build",
+    "prebuild": "rimraf dist",
```

```suggestion
    "prepare": "webpack build",
```

The `prepare` script is no longer needed. And let's just call `webpack build` instead of `npm run build` to make it npm/yarn-agnostic.

---

Done